### DISCUSSION TITLE
Update requirements compliance based on controls status

### DISCUSSION THREAD

**Discussion by ab-smith** [intuitem/ciso-assistant-community#1746] [Ideas]
Now that we have a good foundation for decoupling, we can use the status of applied controls to preset the compliance of specific requirements. Multiple options are possible:
1. from an audit, have a power-up, when triggered, the popup will ask if it should set the requirements to compliant or partially compliant and a double confirmation before updating the status. 
2. from a requirement assessment, have a power up, like "sync to controls status" and would update the individual requirements
3. have it as an x-ray rule that can have a hyperlink to trigger the sync

**Comment by AndrzejRPiotrowski** [comment]
+1

**Comment by Patrick-PDV** [comment]
+1

**Comment by ab-smith** [comment]
Available for a while now thanks to sync to actions capability (the same exist for risk assessment as well)
